### PR TITLE
chore(ci): test prepare-cluster on pr

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: .github/workflows/test-create-customer.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          cache-dependency-path: |
+            **/package-lock.json
+
+      - name: build
+        run: make package-create-customer
+
+      - name: pull helm chart
+        run: |
+          helm pull oci://registry-1.docker.io/bitnamicharts/wordpress --version ${{ env.WORDPRESS_CHART_VERSION }}
+
```
[the remark this answers is not in the document]
Comment (emosbaugh) [reply .github/workflows/test-create-customer.yml:35]: How can I pull from a public repo if username and password are required?

[the remark this answers is not in the document]
Comment (emosbaugh) [reply .github/workflows/test-create-customer.yml:35]: i see. will remove

File: .github/workflows/test-kots-install.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        uses: replicatedhq/compatibility-actions/helm-install@v1
+        with:
+          kubeconfig: ${{ steps.create-cluster.outputs.cluster-kubeconfig }}
+          helm-path: helm
+          registry-username: ${{ github.ref_name }}@example.com
+          registry-password: ${{ steps.create-customer.outputs.license-id }}
+          chart: oci://registry.replicated.com/${{ env.APP_SLUG }}/release__${{ steps.create-release.outputs.release-sequence }}/wordpress
+          name: wordpress
+          namespace: default
+          run-preflights: false
```
[the remark this answers is not in the document]
Comment (emosbaugh) [reply .github/workflows/test-kots-install.yml]: fixed

File: .github/workflows/test-prepare-cluster.yml [hunk 3]
```diff
@@ -57,3 +57,38 @@ jobs:
...
+        run: make package-prepare-cluster
+
+      - name: test prepare-cluster
+        id: prepare-cluster
+        uses: ./prepare-cluster
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          app-slug: replicated-actions-ci # from the Replicated QA team
+          yaml-dir: ./test/kots-install
+          kubernetes-distribution: kind
```
Comment (jdewinne) [inline .github/workflows/test-prepare-cluster.yml:85]: Will this fail on the preflights?

**Review by jdewinne — APPROVED** [review]

**Pull request merged**